index.src.html

```diff
@@ -782,25 +782,60 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
             return;
           }
           if (credential.<a attribute for="Credential">type</a> == "<a const>password</a>") {
-            fetch("https://example.com/loginEndpoint", { credentials: credential, method: "POST" })
+            var form = new FormData();
+            form.append('username_field', credential.id);
+            form.append('password_field', credential.password);
+            var opt = {
+              method: 'POST',
+              body: form,
+              credentials: 'include  // Send cookies.
```

Comment:
Inconsistent quoting of strings also `credentials: 'include'` vs `fetch("https://example.com/loginEndpoint", opt)`